arrow/arrow.py

```diff
@@ -1282,6 +1283,102 @@ def gather_timeframes(_delta: float, _frame: TimeFrameLiteral) -> float:
                 "Please consider making a contribution to this locale."
             )
 
+    def dehumanize(self, timestring: str, locale: str = "en_us") -> "Arrow":
+        """Returns an arrow object that represents the
+        date/time represented by a humanzied arrow time string.
+        """
+
+        # Create current time object
+        current_time = self.fromdatetime(self._datetime)
+
+        # Create an object containing the relative time info
+        time_object_info = {
+            "seconds": 0,
+            "minutes": 0,
+            "hours": 0,
+            "days": 0,
+            "weeks": 0,
+            "months": 0,
+            "years": 0,
+        }
```

Comment:
```suggestion
        time_object_info = dict.fromkeys(["seconds", "minutes", "hours", "days", "weeks", "months", "years"], 0)
```
Purely personal preference, feel free to ignore.